Sources/Utilities/Network/NetworkClientele.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+import Networking
+
+protocol NetworkClientele {
```

Comment:
I feel like this can have a better name.  RequestAuthorizing?  AuthorizedRequester?